addons/mrp/__manifest__.py

```diff
@@ -65,7 +65,11 @@
     ],
     'demo': [
         'data/mrp_demo.xml',
-        'data/mrp_lot_demo.yml'],
+        # 'data/mrp_lot_demo.yml'
```

Comment:
`mrp_demo.xml` has `<function>` tags that later `mrp_lot_demo.yml` uses.
This `<function>` tags only are executed on install, not in updates.
When migrating (migration is an update), this tags are not executed and `mrp_lot_demo.yml` breaks. Thus, `mrp_lot_demo.yml` is disabled to avoid the breaking.

OpenUpgrade is a tool just for migrating, not for testing nor working.

Reply:
No, if you are disabling this, something is not correct. We need to test the migration, and it's done through demo data, so enable it and solve any problem. If not, comment the exact problems you find.

Reply:
You cannot test the demo data that is not loaded. The `<function>` tags in the `mrp_demo.xml` are not loaded!

Reply:
OK, I see this doesn't include demo data. Please add a comment about that.

Reply:
The comment is already added in the manifest...